resources/js/Pages/Results.vue

```diff
@@ -145,11 +145,11 @@
                     [decision.id]: decision
                 };
             },
-            send( item: string ): void {
+            async send( item: string ): Promise<void> {
 
-                if(this.decisions && Object.prototype.hasOwnProperty.call(this.decisions, item)){
-
-                    this.$inertia.put( '/mismatch-review', this.decisions[item] as unknown as RequestPayload );
+                if(this.decisions[item]){
+                    // use axios in order to preserve saved mismatches
+                    axios.put('/mismatch-review', this.decisions[item]);
```

Comment:
yes, I would say so, otherwise this function will throw if the request doesn't succeed, and it will probably take the rest of the app with it. Also, it seems as though we forgot the `await` keyword, which means that right now, the decisions for the item will be deleted without necessarily completing the request successfully (async is lovely isn't it ;) )